src/backEnd.js

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable import/no-mutable-exports */
 
-let tasks = [
-  { description: 'Laundary', completed: true, index: 1 },
-  { description: 'Car wash', completed: false, index: 2 },
-  { description: 'Swimming', completed: false, index: 3 },
-];
+let tasks = [];
+
+const loadTaskList = () => {
+  let loadTasks = JSON.parse(localStorage.getItem('tasks'));
+  if (loadTasks == null) {
+    loadTasks = [];
+  }
+  tasks = loadTasks;
+  return tasks;
+};
```

Comment:
- [ ] When I refresh my server, everything disappears. Please fix the local storage functionality so that every action a user does with the application can be saved.

Reply:
Requested fix implemented. Thank you for your feedback